add-on/_locales/en/messages.json

```diff
@@ -29,7 +29,7 @@
   },
   "panel_statusSwarmPeers": {
     "message": "Peers",
-    "description": "A label in Node status section of Browser Action pop-upi (panel_statusSwarmPeers)"
+    "description": "A label in Node status section of Browser Action pop-up, also used as Welcome ready message button text for opening Web UI peers page (panel_statusSwarmPeers)"
```

Comment:
nit: I was reusing labels like this in the past, but realized it is safer to split and have two keys, one for each use.

Insight: in some locales, people may decide to use slightly different label in each place (either sounds or looks better). By default, if the source label remains the same, it will be automatically translated in both places, so we don't introduce any overhead for translators on Transifex, but makes it more flexible if they need to change one of them.

Please create a new key for each place. :pray:

Reply:
Thanks for settling my mind on this one -- was erring for efficiency but will go with completeness in the future! 😊